src/views/preview/preview.scss

```diff
@@ -228,6 +232,14 @@ $stage-width: 480px;
         flex-shrink: 1;
     }
 
+    .credit-text {
+        /* Line clamping is not supported in IE, FF yet */
+        display: -webkit-box;
+        overflow: hidden;
+        -webkit-box-orient: vertical;
+        -webkit-line-clamp: 2;
```

Comment:
I don't think we should use any styling that only works in one browser because people will begin reporting it as an issue on the browsers that don't support this.  Additionally since this is not standard CSS, there's no guarantee that it will continue working in the future.

There may be a JS library for this feature, but I don't think this is important enough to bring something in to do it. Maybe we should instead just keep the `max-height` of the box to the height of two lines, add `overflow: hidden` and set the `title` of the box to the entire string?

I also wonder if we even want to clip the contents of this box or do we want to let it get as tall as it needs to to display the entire text, no matter what? Is the clipping in @carljbowman's designs?